### ISSUE TITLE
VS Code on Ubuntu 22.04 under Remote Desktop is logging me off

### ISSUE THREAD

**Issue by gfmoore** [microsoft/vscode#179246]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
- OS Version: 

Steps to Reproduce:

1.  I connect to my server using windows Remote Desktop (Windows 10)
2.  I start VS Code and do a few things
3.  VS Code (crashes) and logs me out. 
4. I have to restart the (headless) server to get back to auto logged in and then can use Remote desktop

I haven't yet determined if anything in particular is causing this behaviour. The last thing I did was to close a project folder and was going to try to open another project folder.

This behaviour occurred with insiders so I reverted to stable version

Server: HPE ProLiant MicroServer Gen8 Xeon E3-1220L V2 @ 2.30GHz16GB ECC RAM 12TB (4x3TB) Storage
Ubuntu 22.04  (seems okay with everything else I've been using (Node etc))
Windows 10 Pro 22H2
VS Code Version: 1.77.0
Commit: 7f329fe6c66b0f86ae1574c2911b681ad5a45d63
Date: 2023-03-29T10:05:35.165Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Linux x64 5.19.0-35-generic snap
Sandboxed: Yes

I need some extensions Springboot, Java

**Comment by gfmoore** [comment]
OOPS, this behaviour just occured when I launched the software updater - clearly not a VSCode issue, must be a Remote Desktop issue perhaps.

**Comment by gfmoore** [comment]
Closed

**Comment by gfmoore** [comment]
Nope it does seem that VSCode is at issue. I had closed VS Code when I opened the software updater. On a restart I was using software updater with no issues. Installed insiders, started insiders, opened a folder, closed it and went to re-open a different folder - kicked off.

**Comment by gfmoore** [comment]
I have removed code and insiders using the ubuntu uninstall as I was also getting issues not being able to debig node apps.
I have followed instructions here to install using apt rather than snap. 
https://askubuntu.com/questions/1306455/vscode-does-not-want-to-launch 

At least I can now debug node. I'll report back on stability.

**Comment by gfmoore** [comment]
I managed to get a reproducable use case. I was debugging node and hit a breakpoint, went to open "File" and it logged me off.

I updated to Ubuntu 22.10, same problem.

I am now using Nautilus and no problem so far.

I suspect then that VS Code is not the problem, but that "File" with it is.

I'll close this again and re-open if Nautilus file manager causes similar issue. I hate these hard to figure out bugs :(
